templates/fine-tune-llm_v2/cookbooks/bring_your_own_data/README.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bring your own data \n",
+    "**⏱️ Time to complete**: 10 minutes\n",
+    "\n",
+    "This guide focuses on how you can bring your own data to fine-tune your model on the Anyscale Platform. Make sure you've gone over the [basic fine-tuning guide](../../README.md) before going over this cookbook.\n",
```

Comment:
Create a table of content that user can click and jump to the relevant section.

Reply:
Done!